typescript/test/test_application_js/.gitignore

```diff
@@ -1,3 +1,4 @@
 package-lock.json
 node_modules
 showcase-server
+index_ts.js
```

Comment:
What is this file?

Reply:
it was used to ignore the compiled file for `index_ts.ts`, removed it, thanks